src/dotnet/commands/dotnet-help/HelpCommandParser.cs

```diff
@@ -0,0 +1,25 @@
+﻿using Microsoft.DotNet.Cli.CommandLine;
+
+namespace Microsoft.DotNet.Tools.Help
+{
+    internal static class HelpCommandParser
+    {
+        public static Command Help()
+        {
+            return Create.Command(
+                "help",
+                LocalizableStrings.AppFullName,
+                Accept.ZeroOrOneArgument()
+                    .With(
+                        LocalizableStrings.CommandArgumentDescription,
+                        LocalizableStrings.CommandArgumentName),
+                Create.Option(
+                    "-h|--help|-?|/?",
```

Comment:
Should we even have a help for the help command? If we do, should we display help them for the command itself, instead of the general CLI help? For instance, I don't see where we say that executing `dotnet help add` will open a page with help information for the add command.

cc @richlander